### ISSUE TITLE
Cannot build the binary

### ISSUE THREAD

**Issue by odiroot** [tsmetana/mpk3-settings#1]
Running on Manjaro 21.0.4

Steps I took:

* installed `rtmidi`
* cloned the repo
* ran `./autogen.sh`
* ran `./configure`
* ran `make`

I'm getting this compilation error:
```
gcc -DHAVE_CONFIG_H -I. -I..  -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/lzo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cloudproviders -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/at-spi-2.0 -pthread  -pthread -I/usr/include/rtmidi -D__UNIX_JACK__ -D__LINUX_ALSA__   -Wall -g  -MT file.o -MD -MP -MF .deps/file.Tpo -c -o file.o file.c
file.c: In function ‘file_read’:
file.c:63:88: error: ‘sys_errlist’ undeclared (first use in this function)
   63 |   *err = g_error_new(G_FILE_ERROR, errno_save, "%s: Error opening file: %s", filename, sys_errlist[errno_save]);
      |                                                                                        ^~~~~~~~~~~
file.c:63:88: note: each undeclared identifier is reported only once for each function it appears in
file.c: In function ‘file_write’:
file.c:167:88: error: ‘sys_errlist’ undeclared (first use in this function)
  167 |   *err = g_error_new(G_FILE_ERROR, errno_save, "%s: Error opening file: %s", filename, sys_errlist[errno_save]);
      |                                                                                        ^~~~~~~~~~~
make[2]: *** [Makefile:387: file.o] Error 1
```

**Comment by tsmetana** [comment]
Hi. Thanks for reporting and sorry for the slow response. The `sys_errlist` is a global variable defined in `errno.h` but the manpage suggests it needs also `stdio.h`. In Fedora I seem to have this header included implicitly through the others. Might be this is different in Manjaro. Could you try to add `#include <stdio.h>` at line 6 (above `#include <errno.h>`) in the `src/file.c`? If that would work I can add it too.

**Comment by odiroot** [comment]
Hi @tsmetana This didn't work but I did a dirty thing and replaced `sys_errlist[errno_save]` with silly constant string `"ERRNO_SAVE"` and now it compiles.

**Comment by tsmetana** [comment]
Hmmpf... Thanks. I'll look at it more closely.

**Comment by odiroot** [comment]
Thank you! I'm a bit too rusty with C to debug it myself.

My suspicion is `sys_errlist` is possibly removed from `stdio.h`,

**Comment by tsmetana** [comment]
You're right. After upgrading Fedora I got the same error. This should be fixed in 6f8589c.

**Comment by odiroot** [comment]
Great work. Consider it solved.
